doc/installation/iOS.md

```diff
@@ -117,6 +206,53 @@ If it is installed but you still have the error, maybe Xcode doesn't find it: ru
 
 </details>
 
+<br />
+<details>
+  <summary>No binary rubies available for: osx/13.5/arm64/ruby-3.0.0.
+Continuing with compilation.</summary>
+
+https://stackoverflow.com/questions/42735805/what-does-no-binary-rubies-available-mean
+
+</details>
+
+<br />
+<details>
+  <summary> After the 'bundle exec fastlane ios download_certificates --env testing' command, you gave the SSH git repository link and it's not doing anything.</summary>
+    It might be an issue with your ssh (for exemple if you only cloned the repository through http), try to clone the repository elswhere and try again.
```

Comment:
suggestion: juste pour etre certain d'enlever toute confusion possible

```suggestion
    It might be an issue with your ssh (for exemple if you only cloned the repository through http), try to clone the repository elswhere using ssh and try again.
```